packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -682,6 +707,114 @@ class Viewport implements IViewport {
     return true;
   }
 
+  public resetPanOffset(widthWorld?: number, heightWorld?: number) {
```

Comment:
The basic idea is to set the pan and zoom to [0,0] and 1 after applying the initial resetCamera conditions, as well as
including the initial center and canvas center.